src/Umbraco.Web.Common/Security/BackofficeSecurity.cs

```diff
@@ -49,6 +49,14 @@ public class BackOfficeSecurity : IBackOfficeSecurity
         }
     }
 
+    private Attempt<Guid> GetUserKey()
```

Comment:
It this method superseding `GetUserId()` - and should that then be removed or at least obsoleted?

Reply:
Looking into this, it honestly seems like `GetUserId()` is only used once, and in that, we never use the ID anyways 🤷 
Seems to me like it can be obsoleted 👍